apps/users/routers.py

```diff
@@ -136,7 +134,7 @@ async def delete_user(
 
 @users_router.post("/registration",
                    name="registration",
-                   response_model=JSENDOutSchema[RegistrationOut],
+                   response_model=JSENDOutSchema[CreateUserOut],
```

Comment:
maybe should do inheritance from `CreateUserOut` to `RegistrationOut`, because `CreateUserOut` in registration is not clear

Reply:
The same answer as for the previous one.